examples/README.md

```diff
@@ -20,10 +20,18 @@ The following folders contain our example problem instances:
 
 Identical problem instances across folders have the same properties except that different seed numbers were used to generate them. The instances in the `tuning_instances` folder are used to tune the parameters of the solvers in the package. We then use the parameters to solve the test instances to ensure the performance of the solver is independent of the particular random problem instances generated.
 
-The first line of each instance file contains the following information:
-- the problem instance size
-- the optimum solution
-- whether the solution is optimal (True or False)
-- the seed number used for generating the problem instance in `torch`
+The first line of each instance file contains the following information, in this order:
+- instance size
+- optimum solution (example files used the Gurobi solver to determine these values)
+- best solution (example files used the BFGS solver to determine these values)
+- whether the solution is optimal (`True` or `False`)
+- solution time for Gurobi to solve it 
+- solution time for BFGS to solve it
+- the seed number used for generating the instance in `torch`.
+- number of fractional values in the solution
 
-The second line of the instance file contains the elements of the vector $V$, which defines the instance. The rest of the lines in the file contain the matrix $Q$.
+The second line contains the elements of the vector `V`, which describes the instance.
+
+The rest of the lines before the last line hold the elements of the `Q` matrix.
+
+The final line contains list of additional (optional) information in the problem instance.
```

Comment:
Say "the final line contains the vector of the solution to the problem instance".